content/index.ja.md

```diff
@@ -0,0 +1,700 @@
+
+### コンテンツが見やすく、聞き取りやすくなる {#distinguishable}
+
+識別可能なコンテンツは見やすく、聞き取りやすい。そのようなコンテンツには以下のようなものがある：
+
+-   色が情報を伝えたりコンテンツを識別したりする唯一の方法として使用されていない
+-   デフォルトの前景色と背景色の組み合わせが十分なコントラストを提供している
+-   ユーザーがテキストを400%まで拡大したりテキストの間隔を変更したりしても、情報が失われない
+-   テキストが小さなウィンドウ（"ビューポート"）内で再配置され、ユーザーがテキストを大きくしたときも同様である
+-   テキストの画像がリサイズ可能であり、実際のテキストに置き換えられるか、可能な限り避けられる
+-    ユーザーがウェブサイトで再生される音声を一時停止、停止、または音量を調整できる
+-   背景音は低く設定されているか、干渉や気晴らしを避けるためにオフにできる
+
+この要件を満たすことは、前景と背景を区別し、重要な情報をより識別しやすくするのに役立ちます。これには、支援技術を使用しない人々や、背景の目立つ音声や視覚コンテンツから干渉を受ける可能性がある支援技術を使用する人々への配慮が含まれます。例えば、色覚異常の多くの人々は特定のツールを使用せず、テキストとその周囲の背景の間に十分な色のコントラストを提供する適切なデザインに頼っています。他の人にとっては、自動的に再生される音声がテキスト読み上げや[支援聴取デバイス (ALDs)](/teach-advocate/accessible-presentations/#ald "definition")と干渉する可能性があります。
+
+{% include excol.html type="start" %}
+
+#### 識別可能性に関するアクセシビリティ要件（技術仕様へのリンク）
+{:.no_toc}
+
+{% include excol.html type="middle" %}
+
+**WCAG**
+
+-   [ガイドライン 1.4 - 識別可能](https://www.w3.org/WAI/WCAG22/quickref/#distinguishable)
+
+**UAAG**
+
+-   [ガイドライン 1.3 - ハイライト表示](https://www.w3.org/TR/UAAG20/#gl-interaction-highlight)
+-   [ガイドライン 1.4 - テキスト構成](https://www.w3.org/TR/UAAG20/#gl-text-config)
+-   [ガイドライン 1.5 - 音量構成](https://www.w3.org/TR/UAAG20/#gl-volume-config)
+-   [ガイドライン 1.6 - 合成音声の構成](https://www.w3.org/TR/UAAG20/#gl-speech-config)
+-   [ガイドライン 1.7 - ユーザースタイルシートの構成](https://www.w3.org/TR/UAAG20/#gl-style-sheets-config)
+-   [ガイドライン 1.8 - ビューポート内のオリエンテーション](https://www.w3.org/TR/UAAG20/#gl-viewport-orient)
+-   [ガイドライン 1.9 - 代替ビュー](https://www.w3.org/TR/UAAG20/#gl-alternative-views)
+-   [ガイドライン 1.10 - 要素情報](https://www.w3.org/TR/UAAG20/#gl-info-link)
+
+**ATAG**
+
+-   [原則 A.1: オーサリングツールのユーザーインターフェイスは適用可能なアクセシビリティガイドラインに従う](https://www.w3.org/TR/ATAG20/#principle_a1)
+-   [ガイドライン A.3.7: (オーサリングツールのユーザーインターフェイス用) プレビューが市場に出回っているユーザーエージェントと同等以上にアクセシブルであることを保証する](https://www.w3.org/TR/ATAG20/#gl_a37)
+-   [パート B. アクセシブルなコンテンツの制作をサポートする](https://www.w3.org/TR/ATAG20/#part_b)
+
+{% include excol.html type="end" %}{% include excol.html type="start" %}
+
+#### 区別可能性に関連する記事
+{:.no_toc}
+
+{% include excol.html type="middle" %}
+
+-   [Lee、色覚異常を持つオンラインショッパー](/people-use-web/user-stories/#shopper)
+-   [Martine、聴覚障害を持つオンライン学生](/people-use-web/user-stories/#onlinestudent)
+-   [Ilya、視覚障害を持つ上級職員](/people-use-web/user-stories/#accountant)
+-   [Yun、低視力、手の震え、および軽度の短期記憶障害を持つ退職者](/people-use-web/user-stories/#retiree)
+-   [Kaseem、聴覚障害と視覚障害を持つ10代の若者](/people-use-web/user-stories/#teenager)
+
+{% include excol.html type="end" %}
+
+## 操作可能なユーザー・インターフェースとナビゲーション {#operable}
+
+### キーボードからの操作が可能 {#keyboard}
+
+多くの人々はマウスを使用せず、キーボードに頼ってウェブとやり取りします。これには、フォームコントロール、入力欄、その他のユーザーインターフェイスコンポーネントを含む、すべての機能へのキーボードアクセスが必要です。
+
+キーボードのアクセシビリティ:
+
+-   マウスで利用可能なすべての機能がキーボードでも利用可能である
+-   キーボードフォーカスがコンテンツの任意の部分に閉じ込められない
+-   ウェブブラウザ、オーサリングツール、およびその他のツールがキーボードサポートを提供する
+
+この要件を満たすことで、人間工学的なレイアウトのキーボード、画面上のキーボード、スイッチデバイスなどの代替キーボードを使用する人々を含む、キーボードユーザーが支援されます。また、音声認識（音声入力）を使用してウェブサイトを操作したり、キーボードインターフェースを通じてテキストを入力したりする人々にも役立ちます。
```

Comment:
@takuan517 A shortcut seems to have been taken in the translation.

Original version is 
> Meeting this requirement helps keyboard users, including people **using alternative keyboards** such as keyboards with ergonomic layouts, on-screen keyboards, or switch devices.

"alternative keyboards" seems to be missing in the translation

Reply:
@remibetin 
"Alternative keyboards" is Japanese for "代替キーボード", so I believe it is included in the translation.
Please note that the Japanese translation has been corrected just in case.